addons/l10n_se_ocr/models/res_partner.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api, _
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    use_vendor_ocr = fields.Boolean(string='Use Vendor OCR', help='This Vendor uses OCR Number on their Vendor Bills.')
```

Comment:
Since this is used in the checkup on invoice to verify if the vendor uses a OCR number or not:

From account_move.py, line 72-79, see third if statement.
```
@api.onchange('invoice_payment_ref')
    def _onchange_invoice_payment_ref(self):
        if hasattr(super(AccountMove, self), "_onchange_payment_ref"):
            super(AccountMove, self)._onchange_payment_ref()
    
        """ If Vendor Bill and Payment Reference is changed check validation. """
        if self.partner_id and self.type == 'in_invoice' and self.partner_id.use_vendor_ocr and not self.env['l10n_se.luhn'].verify(str(self.invoice_payment_ref)):
            return {'warning': {'title': _('Warning'), 'message': _('Vendor require OCR Number as payment reference. Payment reference isn\'t a valid OCR Number.')}}
```

Reply:
Ok, I get it again.  It is ok, we can keep it, but the label should be "Check Vendor OCR" then.  And I would only use it for this.  It is confusing with what is happening in the other onchange:  if ... self.partner_id.use_vendor_ocr and self.partner_id.default_vendor_ocr.  It think we should remove the self.partner_id.user_vendor_ocr over there then.

Reply:
Do you want the label of the field as check_vendor_ocr on res_partner instead of use_vendor_ocr?

Reply:
l10n_se_check_vendor_ocr